### ISSUE TITLE
Dependency modules?

### ISSUE THREAD

**Issue by arthabus** [mooz/node-pdf-image#59]
> Ensure you have `convert`, `gs`, and `pdfinfo` (part of poppler) commands.

Could you please list exact modules to install here?

**Comment by sonawanevip** [comment]
> > Ensure you have `convert`, `gs`, and `pdfinfo` (part of poppler) commands.
> 
> Could you please list exact modules to install here?

hi, did you find any answer for this? if yes please share with us

**Comment by arthabus** [comment]
@sonawanevip nope... I ended up using ghostscript directly as it does what I need.

Not sure if that might be relevant, but here is the article I ended up following that explains how to install ghostscript in node if you are using firebase:

https://tech.residebrokerage.com/using-firebase-cloud-functions-with-ghostscript-for-pdf-to-image-conversion-f81b248d3b22

I guess something similar needs to be done for all packages for this library to work - `convert` must be from ImageMagic package, `gs` is ghostscript and `pdfinfo` from poppler
